### DISCUSSION TITLE
Smartlife Scenes do not appear in Homekit

### DISCUSSION THREAD

**Discussion by mpferreiraf** [milo526/homebridge-tuya-web#560] [General]
I have many Smartlife Scenes, both Automation or Tap to Run, does not appear in the Homekit when all scenes are selected. When I force white list scenes to appear, they don’t work. Does anyone have this problem ?

**Comment by milo526** [comment]
Do you get any logs in homebridge?

**Comment by mpferreiraf** [comment]
Hi I don’t know how to obtain that. Enviado do meu iPhoneEm 26 de ago. de 2023, à(s) 15:06, Milo Cesar ***@***.***> escreveu:﻿
Do you get any logs in homebridge?

**Comment by mpferreiraf** [comment]
HiWhat I realized I that all Smartlife Automation came to HomeKit but not the Tap-to-Run ones. Enviado do meu iPhoneEm 26 de ago. de 2023, à(s) 15:30, Milton Hotmail ***@***.***> escreveu:﻿
Hi I don’t know how to obtain that. Enviado do meu iPhoneEm 26 de ago. de 2023, à(s) 15:06, Milo Cesar ***@***.***> escreveu:﻿
Do you get any logs in homebridge?

**Comment by mpferreiraf** [comment]
Is there any limit for the number of devices or scenes ?Enviado do meu iPhoneEm 26 de ago. de 2023, à(s) 15:06, Milo Cesar ***@***.***> escreveu:﻿
Do you get any logs in homebridge?
